tflog/provider_test.go

```diff
@@ -29,6 +30,7 @@ func TestWith(t *testing.T) {
 				{
 					"@level":        hclog.Trace.String(),
 					"@message":      "test message",
+					"@module":       logging.DefaultProviderRootLoggerName,
```

Comment:
I'm from the "school of thought" that test _fixtures_ should be hardcoded and explicit.

IMHO here you should have the same string `"provider"` duplicated a bunch of times, because using the `const` itself is helping reducing the chance of the test breaking if, for example, one introduces a typo in the value of `DefaultProviderRootLoggerName` above.

Reply:
Updated everywhere.